.travis.yml

```diff
@@ -3,8 +3,17 @@ node_js:
 - '8'
 after_success:
 - npm run coveralls
+before_install:
+- cd ..
+- git clone https://github.com/jomadoye/openmrs-react-components.git openmrs-react-components
```

Comment:
This is temporal the `openmrs-react-components` would soon be published to npm, and this file would be updated to reflect it.

Reply:
Thanks for figuring this out @jomadoye.... can we just change this to reference openmrs-react-components in the main OpenMRS project, ie:

https://github.com/openmrs/openmrs-react-components.git